src/components/profilePage/RatingStar/RatingStarStyles.css

```diff
@@ -0,0 +1,20 @@
+.header{
+    font-style: normal;
+}
+#star{
```

Comment:
Don't use id's for this kind of jobs. use class  instead and follow the convention.